lighthouse-core/audits/seo/is-crawlable.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+const BLOCKLIST = new Set([
+  'noindex',
+  'none',
+]);
+const ROBOTS_HEADER = 'x-robots-tag';
+const UNAVAILABLE_AFTER = 'unavailable_after';
+
+/**
+ * Checks if given directive is a valid unavailable_after directive with a date in the past
+ * @param {string} directive
+ * @returns {boolean}
+ */
+function isUnavailable(directive) {
+  const parts = directive.split(':');
```

Comment:
Yeah, I agree that'd be much more elegant, but in this case it won't work:

```javascript
const [key, value] = 'unavailable_after: 12 Jun 2017 12:30:00'.split(':');
```
value in this case would be ` 12 Jun 2017 12` instead of ` 12 Jun 2017 12:30:00`. I could do:
```javascript
const [key, ...value] = 'unavailable_after: 12 Jun 2017 12:30:00'.split(':');
```
But then, `value` is an array and I still have to `.join(':')` it, so not much different from a current solution :(

Reply:
Would `split(':', 1)` resolve your concern?

Reply:
TIL about second parameter of `.split`!
This gives me access to `unavailable_after` but doesn't give me the date:

```javascript
const [key, value] = 'unavailable_after: 12 Jun 2017 12:30:00'.split(':', 1);
```

`value` will be empty. Am I missing something here? 🤔

Reply:
Yeah you're right, it doesn't do what I thought it would. Carry on!